config/default.yml

```
@@ -215,6 +215,12 @@ Performance/RedundantSortBlock:
   Enabled: 'pending'
   VersionAdded: '1.7'
 
+Performance/RedundantSplitRegexpArgument:
+  Description: 'This cop identifies places where `split` argument can be replaced from a deterministic regexp to a string.'
+  SafeAutoCorrect: true
+  Enabled: true
```

Can you update enabled status to pending?

```suggestion
  Enabled: pending
```

https://docs.rubocop.org/rubocop/1.2/versioning.html#pending-cops